config/environments/development.rb

```diff
@@ -45,7 +45,19 @@
     protocol: 'http'
   }
 
-  config.action_mailer.delivery_method = ENV['USER'] == 'vagrant' ? :letter_opener_web : :letter_opener
+  if ENV['USER'] == 'vagrant'
+    # When running in a vagrant VM we can't use the letter_opener since it
+    # requires access to the browser exectuable.
+    config.action_mailer.delivery_method = :letter_opener_web
+  elsif ENV['USER'].nil?
+    # With running is docker we can't even use letter_open_web because
+    # sidekiq runs in its own container which doesn't share a ./tmp direcotry
+    # with the main web server.
```

Comment:
I think we can get rid of all of this above. We're not supporting Vagrant, and now that we're using Mailcatcher, we can get rid of letter opener as well.